test/unit/carriers/new_zealand_post_test.rb

```diff
@@ -145,7 +145,6 @@ def test_international_empty_json_response_error
     @carrier.expects(:commit).returns([""])
     error = @carrier.find_rates(@wellington, @ottawa, package_fixtures[:book]) rescue $!
     assert_equal ActiveShipping::ResponseError, error.class
-    assert_equal "A JSON text must at least contain two octets!", error.message
```

Comment:
This error message is tied to the version of JSON being used. And fails across the 1.x 2.x version change.